Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'turbolinks'
 
 # View
 gem 'simple_form'
+gem 'enumerize'
```

Comment:
@nicholasjhenry My understanding of the issue with the `enumerize` gem was that it was _duplicated_ in the Gemfile. So I fixed it a few days ago ([here is the relevant commit](https://github.com/cawel/Montreal.rb/commit/ac85398e4157ca02d14a0c4ad528c7130fdbf722])), changing the Gemfile to include it only once.

Both my `migrations-cleanup` branch and my `pages-cleanup` branch have that gem only once in the Gemfile. I'm not sure about what you expect from me. Is there something I misunderstood?